rules/const.bzl

```diff
@@ -285,3 +285,8 @@ EARLGREY_LOC_ALERTS = [
     "loc_dummy14",
     "loc_dummy15",
 ]
+
+KEY_AUTHENTICITY = [
```

Comment:
We should update `sw/device/silicon_creator/rom/keys/README.md` or any other relevant documents to describe how this works. There are downstream integrators trying to reproduce this as part of their post-silicon prep plan.

Downstream integrators are usually not familiar enough with the codebase to poke around Bazel config files.

Reply:
Created #18784